spec/test_app/app/controllers/basechurch/v1/bulletins_controller_spec.rb

```diff
@@ -99,7 +120,7 @@
     it_behaves_like 'a response containing a bulletin'
   end
 
-  describe 'POST /:group_slug/bulletins' do
+  describe 'POST /bulletins' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.